entity-framework/core/get-started/index.md

```diff
@@ -18,11 +18,11 @@ You can follow the tutorial by using Visual Studio on Windows, or by using the .
 
 Install the following software:
 
-# [.NET Core CLI](#tab/netcore-cli)
+## [.NET Core CLI](#tab/netcore-cli)
```

Comment:
Wasn't aware, thanks!

Reply:
This also shows warning. H2 with same title multiple times. But multiple H1 is more severe error. I will also test out not put it as heading at some point.

Reply:
The issue here is probably that the tab syntax is a docfx extension, this is just how it works... It's weird for multiple H2 with the same title to flag an issue even without tabs, maybe we can suppress certain issues.